script/DeployAllSandbox.s.sol

```diff
@@ -165,4 +160,17 @@ contract DeployAllSandboxScript is Script {
 
         vm.stopBroadcast();
     }
+
+    function deployPaymentToken() internal returns (MockToken usdc, MockToken usdt, MockToken usdce) {
```

Comment:
Call this per token, rather than returning three tokens. Each should have a different version.